ooniapi/services/ooniprobe/src/ooniprobe/routers/v1/probe_services.py

```diff
@@ -0,0 +1,126 @@
+import logging
+from datetime import datetime, timezone, timedelta
+import time
+from typing import List
+
+from fastapi import APIRouter, Depends, HTTPException, Response
+
+from ...common.dependencies import get_settings
+from ...common.routers import BaseModel
+from ...common.auth import create_jwt, decode_jwt, jwt
+from ...common.config import Settings
+from ...common.utils import setnocacheresponse
+
+router = APIRouter(prefix="/v1")
+
+log = logging.getLogger(__name__)
+
+class ProbeLogin(BaseModel):
+    # Allow None username and password 
+    # to deliver informational 401 error when they're missing
+    username : str | None = None
+    # not actually used but necessary to be compliant with the old API schema
+    password : str | None = None 
+
+class ProbeLoginResponse(BaseModel):
+    token : str
+    expire : str
+
+@router.post("/login", tags=["ooniprobe"], response_model=ProbeLoginResponse)
+def probe_login_post(
+    probe_login : ProbeLogin, 
+    response : Response,
+    settings : Settings = Depends(get_settings),
+) -> ProbeLoginResponse:
+    global log
```

Comment:
Is it necessary to make the `log` variable global?

Reply:
Not actually, I will remove it. I was mixing it with the pattern used in the monolith ([example here](https://github.com/ooni/backend/blob/9fa38c97c9f614e94a830234955c1aa0fe9157bc/api/ooniapi/citizenlab.py#L572)), but now that I look closer is very different